x/auth/module.go

```diff
@@ -0,0 +1,64 @@
+package auth
+
+import (
+	"encoding/json"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	abci "github.com/tendermint/tendermint/abci/types"
+)
+
+// name of this module
+const ModuleName = "auth"
+
+// app module object
+type AppModule struct {
+	accountKeeper AccountKeeper
+}
+
+// NewAppModule creates a new AppModule object
+func NewAppModule(accountKeeper AccountKeeper) AppModule {
+	return AppModule{
+		accountKeeper: accountKeeper,
+	}
+}
+
+var _ sdk.AppModule = AppModule{}
+
+// module name
+func (AppModule) Name() string {
+	return ModuleName
+}
+
+// register invariants
+func (AppModule) RegisterInvariants(_ sdk.InvariantRouter) {}
+
+// module message route name
+func (AppModule) Route() string { return "" }
```

Comment:
Isn't this supposed to be (or something similar):

```go
func (AppModule) Route() string { return TransactionRoute }
```

Reply:
There are no routes, but this function still need to exist to fulfill the `AppModule` interface